src/index.js

```diff
@@ -1,7 +1,369 @@
+          onClick={() => props.move("east")}
+        >
+          <RightArrow />
+        </button>
+        <button
+          id="south-button"
+          className="nav-button"
+          onClick={() => props.move("south")}
+        >
+          <DownArrow />
+        </button>
+        <button
+          id="west-button"
+          className="nav-button"
+          onClick={() => props.move("west")}
+        >
+          <LeftArrow />
+        </button>
+      </div>
+    );
+  } else {
+    return (
+      <div id="button-wrapper">
+        <button
+          id="north-button"
+          className="nav-button"
+          onClick={() => props.move("north")}
+          disabled
+        >
+          <UpArrow />
+        </button>
+        <button
+          id="east-button"
+          className="nav-button"
+          onClick={() => props.move("east")}
+          disabled
+        >
+          <RightArrow />
+        </button>
+        <button
+          id="south-button"
+          className="nav-button"
+          onClick={() => props.move("south")}
+          disabled
+        >
+          <DownArrow />
+        </button>
+        <button
+          id="west-button"
+          className="nav-button"
+          onClick={() => props.move("west")}
+          disabled
+        >
+          <LeftArrow />
+        </button>
+      </div>
+    );
+  }
+};
+
+const GameButtons = props => {
+  if (props.gameStarted) {
+    return (
+      <div id="button-row">
+        <button disabled id="start-button">
+          Start
+        </button>
+        <button id="guess-button" onClick={props.showGuessForm}>
+          Guess!
+        </button>
+        <button id="quit-button" onClick={props.giveUp}>
+          Give up
+        </button>
+      </div>
+    );
+  } else {
+    return (
+      <div id="button-row">
+        <button id="start-button" onClick={props.startGame}>
+          Start
+        </button>
+        <button disabled id="guess-button">
+          Guess!
+        </button>
+        <button disabled id="quit-button">
+          Give up
+        </button>
+      </div>
+    );
+  }
+};
+
+//Add to the NavBar
+const Nav = props => {
+  return <div id="navbar">Geo-Vermonter</div>;
+};
+
+const InfoBox = props => {
+  if (props.gameOver) {
+    return (
+      <h3>
+        Lat: {props.markerPosition.lat.toFixed(3)} Lon:{" "}
+        {props.markerPosition.lon.toFixed(3)}
+      </h3>
+    );
+  } else {
+    return <h3>Lat: ??? Lon: ???</h3>;
+  }
+};
+
+const GuessForm = props => {
+  if (props.readyToGuess) {
+    return (
+      <form onSubmit={props.handleSubmit}>
+        <label>
+          Which county did you start in:
+          <input
+            id="guess-field"
+            type="text"
+            value={props.guess}
+            onChange={props.handleChange}
+          />
+        </label>
+        <input id="submit-button" type="submit" value="Submit" />
+      </form>
+    );
+  } else {
+    return null;
+  }
+};
```

Comment:
I would like to see these functional components split into their own files for organizational purposes. That's just personal preference though.